src/MixClient.cpp

```diff
@@ -80,6 +82,9 @@ MixClient::MixClient(string const& _dbPath):
 	m_postSeal(Block::Null),
 	m_dbPath(_dbPath)
 {
+	dev::eth::Ethash::init();
+	dev::eth::BasicAuthority::init();
```

Comment:
just to be sure. (there was includes in header) 
we could remove it if you think it is not needed